src/components/common/ColonyActions/ActionDetailsPage/DefaultMotion/MotionPhaseWidget/StakingWidget/StakingInput/StakingControls/StakingControls.tsx

```diff
@@ -26,11 +36,17 @@ const StakingControls = ({ limitExceeded }: StakingControlsProps) => {
       <StakeButton
         isLoadingData={isLoadingData}
         enoughTokensToStakeMinimum={enoughTokensToStakeMinimum}
+        enoughReputationToStakeMinimum={enoughReputationToStakeMinimum}
```

Comment:
The error messages introduced here don't have a lot to do with the form value, therefore we can't use the validation schema to handle error state. 

Without significant reengineering of this widget, I think passing a bunch of props is the simplest way to correctly handle the stake button's disabled stake.

Reply:
Don't worry about, most likely this'll get changed along with the new UX. It's good enough for now.